ktor-server/ktor-server-test-suites/jvm/src/io/ktor/server/testing/suites/ConfigTestSuite.kt

```diff
@@ -57,6 +56,8 @@ abstract class ConfigTestSuite(val engine: ApplicationEngineFactory<*, *>) {
             server.stop(0, 100, TimeUnit.SECONDS)
         }
 
-        assertTrue("Stop time is $time", time < 100)
+        assert(time < 100) {
```

Comment:
Could you tell me why raw assert instead of assertTrue is used?

Reply:
Oh I meant to replace with `kotlin.test` which just has the arguments swapped, I'll fix.